spring-rabbit/src/main/java/org/springframework/amqp/rabbit/support/micrometer/RabbitListenerObservation.java

```diff
@@ -63,14 +60,36 @@ public enum ListenerLowCardinalityTags implements KeyName {
 		 * Listener id.
 		 */
 		LISTENER_ID {
-
 			@Override
 			public String asString() {
 				return "spring.rabbit.listener.id";
 			}
 
-		}
+		},
+
+		/**
+		 * The exchange the listener is plugged to (empty if default exchange).
+		 * @since 3.2
+		 */
+		EXCHANGE {
+			@Override
+			public String asString() {
+				return "messaging.destination.name";
+			}
+
+		},
+
+		/**
+		 * The routing key the listener is plugged to.
+		 * @since 3.2
+		 */
+		ROUTING_KEY {
+			@Override
+			public String asString() {
+				return "messaging.rabbitmq.destination.routing_key";
```

Comment:
This is totally wrong for consumer side.
See the [messaging.rabbitmq.message.delivery_tag](https://opentelemetry.io/docs/specs/semconv/attributes-registry/messaging/) instead.
Can be set into a `context.getCarrier().getMessageProperties().getDeliveryTag()`.

Reply:
done